packages/angular_devkit/core/node/fs.ts

```diff
@@ -5,34 +5,36 @@
  * Use of this source code is governed by an MIT-style license that can be
  * found in the LICENSE file at https://angular.io/license
  */
-import * as fs from 'fs';
+import { statSync } from 'fs';
 
-
-export function isFile(filePath: string): boolean {
-  let stat;
-  try {
-    stat = fs.statSync(filePath);
-  } catch (e) {
-    if (e && (e.code === 'ENOENT' || e.code === 'ENOTDIR')) {
-      return false;
+export namespace fs {
```

Comment:
We need a bigger discussion as `export namespace ...` is not considered best practices. See https://www.typescriptlang.org/docs/handbook/namespaces-and-modules.html#needless-namespacing